tox.ini

```diff
@@ -12,10 +12,12 @@ install_command = pip install {opts} {packages}
 deps =
   flake8
   antsibull-changelog
+  yamllint
 commands =
   flake8 plugins {posargs}
   antsibull-changelog lint
   ./check-ignores-order
+  yamllint -c .yamllint.yaml tests
```

Comment:
Why only check the tests-folder and not everything?

Reply:
tests contains most of the yaml stuff.